mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -85,10 +85,12 @@
 import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListMachineUsersResponse;
 import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListResourceAssigneesRequest;
 import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListResourceAssigneesResponse;
-import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListServicePrincipalCloudIdentitiesRequest;
-import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListServicePrincipalCloudIdentitiesResponse;
 import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListRolesRequest;
 import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListRolesResponse;
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ListServicePrincipalCloudIdentitiesRequest;
```

Comment:
the import order in your intellij is somehow different that ours please check

Reply:
Pardon me but my ordering seems alright. All the imports are now in the correct lexicographical order. Why would `ListServicePrincipalCloudIdentitiesResponse` come before `ListRolesRequest`? Same below, `com.sequenceiq.thunderhead.*` seems to be greater than `com.sequenceiq.cloudbreak.*`, so the former ones shall probably come later in the import list. Could you please clarify what this should look like?